lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -437,47 +433,83 @@ def translate_new_secondary_disabilities(disabilities)
         disabilities
       end
 
+      # Map 'NEW' type disability to EVSS required disability fields
+      #
+      # @param input_disability [Hash] The newly submitted disability
+      # @option input_disability [String] :condition The name of the disability
+      # @option input_disability [String] :classificationCode Optional classification code
+      # @option input_disability [Array<String>] :specialIssues Optional list of associated special issues
+      # @option input_disability [String] :primaryDescription The disabilities description
```

Comment:
I find the wording of this comment confusing — I think the code makes it clear that `condition` et al come from `input_disability`, so having `input_disability` in each line adds noise. Also, what is the usefulness of `@param` and `@option` here? I think the most important thing for this comment to convey is the data type and function of each variable. What if we did something like this?
```
# input_disability [Hash] The newly submitted disability
# condition [String] The name of the disability
# classificationCode [String] Optional classification code
etc
```

Reply:
The reason keys like `@param` and `@option` are used is because thats part of the Yard documentation technique. The same goes for using `input_disability` in each of the options since it ties it back to the original hash param.